wizelineacademy/laboratoria-hacker-books
# Feat/laboratoria fe 1/navbar component

## Component Navbar

Created component navbar and first work with tailwind styles and storybook for show component base wireframe.

## Testing instructions

- [ ] [1] Pull change branch Feat/laboratoria-FE-1/navbar-component

- [ ] [2] Review styles component navbar file navbar must content storybook and component navbar

:checkered_flag:

## How should this be manually tested?

- Check that component base wireframe and view like Navbar. 

-  In code, you will find file navbar, complement svg, and component for this

-  This Navbar will be separate some elements like search, new link, and profile in other tasks

:construction:

## Screenshots - wireframe captures

![Screen Shot 2020-11-10 at 23 10 26](https://user-images.githubusercontent.com/56928486/98771828-e945a480-23aa-11eb-961c-e7aa92d55365.png)

![Screen Shot 2020-11-10 at 23 09 59](https://user-images.githubusercontent.com/56928486/98771818-e480f080-23aa-11eb-8b96-001b93c300f2.png)

:camera:

## Testing

Please could you help with this PR 
@jonathanperazawl @ivanovishado 

:question:
